crates/pink-libs/subrpc/src/objects.rs

```diff
@@ -82,9 +82,11 @@ pub struct TransactionResponse<'a> {
 #[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
 pub struct ExtraParam {
     // 0 if Immortal, or Vec<u64, u64> for period and the phase.
-    era: Era,
+    pub era: Option<Era>,
     // Tip for the block producer.
-    tip: u128,
+    pub tip: Option<u128>,
```

Comment:
Looks like it doesn't need to turn the `era` and `tip` into `Option<T>`, just add `#[derive(Default)]` on the ExtraParam,
because the user side just uses the default value if they are None: `let tip = extra.tip.unwrap_or(0);`

Reply:
Nice suggestion, done